### PR TITLE
Fix com media layout

### PR DESCRIPTION
**Pull request by eopws**
Pull Request for Issue #33737

### Summary of Changes

Fixed little mistakes in layout of media component

### Testing Instructions

Go to administrator > media component and switch to list format

### Actual result BEFORE applying this Pull Request

You cannot open image preview by double click.

On image preview the close button is far away from the image which looks weird
before

### Expected result AFTER applying this Pull Request

You are able to open image preview by double click.

The close button is close to the image which I believe looks more beautiful
after

### CODE REVIEW

**Comment by Kostelano** [comment]
Sorry, but this does not solve the problem I described. Can you showcase PR work online?

**Comment by richard67** [comment]
@Kostelano It needs to run npm ci after applying the patch.

**Comment by eopws** [comment]
> Sorry, but this does not solve the problem I described. Can you showcase PR work online?

I have tested all proposed changes and they work for me

**Comment by richard67** [comment]
@eopws Can it be that you referred to the wrong issue in the description? It seems to be something completely different.

**Comment by eopws** [comment]
> @eopws Can it be that you referred to the wrong issue in the description? It seems to be something completely different.

Probably the description of my PR is too poor.
In issue #33737 @Kostelano says that by double clicking on image in list format the image gets selected, when it should be opened in preview. Then he says that the image icon should be centered. The pull request is created to fix those problems. Sorry,  if something is wrong.

**Comment by richard67** [comment]
> In issue #33737 @Kostelano says that by double clicking on image in list format the image gets selected, when it should be opened in preview. Then he says that the image icon should be centered. The pull request is created to fix those problems. Sorry, if something is wrong.

@eopws I see. I was confused a bit by the screenshot with the red circle around the "X" button. That seems to be a different thing on a first look.

**Comment by Kostelano** [comment]
@richard67 @eopws Thanks, it really works now.

About centering the icon - I'm not sure which is better now. It probably makes sense to place the icon not in the center, but at the level of the folder / image header. Like that:

Screenshot_1

Although, of course, how many people - so many opinions :).

Let's wait for what others have to say.

**Comment by eopws** [comment]
> @richard67 @eopws Thanks, it really works now.
> 
> About centering the icon - I'm not sure which is better now. It probably makes sense to place the icon not in the center, but at the level of the folder / image header. Like that:
> 
> Screenshot_1
> 
> Although, of course, how many people - so many opinions :).
> 
> Let's wait for what others have to say.

It's pretty easy to change. We just have to change the value of vertical-align property.

**Comment by eopws** [comment]
> I see. I was confused a bit by the screenshot with the red circle around the "X" button. That seems to be a different thing on a first look.

I hope it's okay that I didn't make them in different PRs, isn't it?

**Comment by richard67** [comment]
> I hope it's okay that I didn't make them in different PRs, isn't it?

It's ok. I just did not get that it belongs to the issue, too.

**Comment by hans2103** [comment]
I have tested this item :white_check_mark: successfully on 453863eb759292fac10660373b937327128179b7<br /><br />I've ran 'npm ci' after applying the patch
Changed from tile to list view
Double clicked an item and it opens.
The x to close has a better spot now. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

**Comment by hans2103** [comment]
I have tested this item :white_check_mark: successfully on 453863eb759292fac10660373b937327128179b7<br /><br />I've ran 'npm ci' after applying the patch
Changed from tile to list view
Double clicked an item and it opens.
The x to close has a better spot now. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

**Comment by Kostelano** [comment]
@eopws please "pull up" the icons to the same level as the headings and I will mark the PR as successful.

It is, but the inconsistency of the icon-title haunts :). Thank you.

**Comment by eopws** [comment]
> @eopws please "pull up" the icons to the same level as the headings and I will mark the PR as successful.
> 
> It is, but the inconsistency of the icon-title haunts :). Thank you.

I'm done. You can check this.

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on be97076f23ff85457f54c7ecb975cce56c6055bb<br /><br />Thank you, it works!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

File: build/media_source/com_media/scss/components/_media-browser.scss [hunk 1]
```diff
@@ -311,7 +311,7 @@
     position: relative;
     z-index: 0;
     width: 50px;
-    padding: 0;
+    padding: 0.6rem 0;
```
Comment (richard67) [inline build/media_source/com_media/scss/components/_media-browser.scss]:
```suggestion
    padding: .6rem 0;
```
Please fix this so the scss style linter is happy and not reports "Unexpected leading zero" anymore.

Comment (richard67) [reply build/media_source/com_media/scss/components/_media-browser.scss]: I allow myself to commit my suggestion.

**Comment by richard67** [comment]
I've restored the previous test result in the issue tracker so it's properly counted again, because the commit which invalidated it was just a code style change.

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 91009f3e34c2a55b0b988fa81bb80c73bdb52f6e<br /><br />Close button reasonably place. A bit strange that the whole row is clickable - when the pointer is a finger you expect to click once. I guess that is a separate issue.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

**Comment by Quy** [comment]
Filename overlaps close icon.

**Comment by brianteeman** [comment]
The x must be on a separate line to the title

**Comment by ceford** [comment]
> The x must be on a separate line to the title

I think standard practice is to have a modal dialog close button on the same line as the title. I would commit this one because it solves the double click problem. It is really the modal dialog that needs some work to revise the layout.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

File: build/media_source/com_media/scss/components/_media-modal.scss [hunk 2]
```diff
@@ -62,8 +62,8 @@
 
 .media-preview-close {
   position: absolute;
-  bottom: 100%;
-  left: 100%;
+  top: -1%;
```
Comment (infograf768) [inline build/media_source/com_media/scss/components/_media-modal.scss]: Any reason to not change `top: -1%;` to `top: -2rem;` ?

**Comment by eopws** [comment]
> Filename overlaps close icon.

Can filename have more than 1 line?

File: build/media_source/com_media/scss/components/_media-modal.scss (hunk 2 above)
Comment (eopws) [reply build/media_source/com_media/scss/components/_media-modal.scss]: @infograf768 Have you tested `top: -2rem;`?

**Comment by eopws** [comment]
@Quy 
> Filename overlaps close icon.

Changed. Please, test new patch.

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on 6c926b64c074a9796da373993d19f1274e5dd62c<br /><br />Real retest, it works. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6c926b64c074a9796da373993d19f1274e5dd62c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33753">issues.joomla.org/tracker/joomla-cms/33753</a>.</sub>

**Comment by richard67** [comment]
Thanks!

File: build/media_source/com_media/scss/components/_media-modal.scss (hunk 2 above)
Comment (infograf768) [reply build/media_source/com_media/scss/components/_media-modal.scss]: Yep.

**Pull request merged**